react/features/base/util/configUtil.js

```diff
@@ -33,32 +37,43 @@ function getRoomName () { // eslint-disable-line no-unused-vars
 
 /**
  * Parses the parameters from the URL and returns them as a JS object.
- * @param source {string} values - "hash"/"search" if "search" the parameters
- * will parsed from location.search otherwise from location.hash
- * @param dontParse if false or undefined some transformations
- * (for parsing the value as JSON) are going to be executed
+ *
+ * @param {string} source - Values - "hash"/"search" if "search" the parameters
+ * will parsed from location.search otherwise from location.hash.
+ * @param {boolean} dontParse - If false or undefined some transformations
+ * (for parsing the value as JSON) are going to be executed.
+ * @returns {Object}
  */
-// eslint-disable-next-line no-unused-vars
-function getConfigParamsFromUrl(source, dontParse) {
-    var paramStr = (source === "search")? location.search : location.hash;
-    if (!paramStr)
+export function getConfigParamsFromUrl(
+        source: string, dontParse: boolean): Object {
+    let paramStr = source === 'search' ? location.search : location.hash;
+
+    if (!paramStr) {
         return {};
+    }
+
     paramStr = paramStr.substr(1);
-    var result = {};
-    paramStr.split("&").forEach(function (part) {
-        var item = part.split("=");
-        var value;
+    const result = {};
+
+    paramStr.split('&').forEach(part => {
+        const item = part.split('=');
+        let value;
+
         try {
-            value = (dontParse)? item[1] : JSON.parse(
-                decodeURIComponent(item[1]).replace(/\\&/, "&"));
+            value = dontParse ? item[1] : JSON.parse(
+                decodeURIComponent(item[1]).replace(/\\&/, '&'));
         } catch (e) {
-            console.warn("Failed to parse URL argument", e);
-            if(window.onerror)
-                window.onerror("Failed to parse URL argument", null, null,
+            console.warn('Failed to parse URL argument', e);
```

Comment:
We can't use logger here because this file is used for `do_external_connect.js` and logger is not defined there.